src/app/starredentries/starredentries.component.html

```diff
@@ -95,31 +227,69 @@ <h3 class="starred-header">Starred Organizations</h3>
+                          <ng-template #placeholder><span style="height: 50px"></span></ng-template>
+                          <div *ngIf="org?.starredUsers.length !== 0" fxLayout="row" fxLayoutAlign="end center">
+                            <app-organization-starring
+                              [organization]="org"
+                              (stargazersChange)="organizationStarGazersChange()"
+                            ></app-organization-starring>
+                          </div>
+                        </div>
+                        <div data-cy="orgName">
+                          <h5 class="text-break">{{ org.displayName }}</h5>
+                          <span class="org-topic">{{ org.topic }}</span>
+                        </div>
+                      </div>
+                    </a.no-underline>
+                    <div>
+                      <div fxLayout="column" fxLayoutAlign="space-between start" class="info">
+                        <span fxLayout="row" fxLayoutAlign="start center" *ngIf="org?.link">
+                          <mat-icon class="info-icon">link</mat-icon>
+                          <span class="info-overflow"
+                            ><a [href]="org.link" target="_blank" rel="noopener noreferrer">{{ org.link }}</a></span
+                          >
+                        </span>
+                        <span fxLayout="row" fxLayoutAlign="start center" *ngIf="org?.location">
+                          <mat-icon class="info-icon">location_on</mat-icon>
+                          <span class="info-overflow">{{ org.location }}</span>
+                        </span>
+                        <span fxLayout="row" fxLayoutAlign="start center" *ngIf="org?.email">
+                          <mat-icon class="info-icon">email</mat-icon>
+                          <span class="info-overflow"
+                            ><a [href]="'mailto:' + org.email" target="_top">{{ org.email }}</a></span
+                          >
+                        </span>
+                      </div>
+                    </div>
+                  </div>
+                  <div>
+                    <hr class="mb-4" />
+                    <a routerLink="/organizations/{{ org.name }}" class="no-underline" target="_self">
+                      <div fxLayout="row" fxLayoutAlign="space-between center">
+                        <span class="gray-label-tag">
+                          {{ org.collectionsLength }}
+                          <span *ngIf="org.collectionsLength != 1; else oneCollection"> Collections</span>
+                          <ng-template #oneCollection> Collection</ng-template>
+                        </span>
+                        <button mat-raised-button color="accent" class="view-button">View</button>
+                      </div>
+                    </a>
+                  </div>
+                </div>
+              </mat-card>
```

Comment:
this looks copy pasted.  "Limited duplicate code" of https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist

Create a new tiny component if necessary